src/IssueViz.UnitTests/Selection/AnalysisIssueSelectionServiceTests.cs

```diff
@@ -274,41 +322,83 @@ public void SelectedIssueChanged_ChangesSelectedFlow()
             testSubject.SelectedFlow.Should().BeNull();
 
             // Set issue to different value
-            var secondIssue = GetIssueWithFlow(out var secondIssueFirstFlow);
+            var secondIssueFirstFlow = CreateFlow();
+            var secondIssue = CreateIssue(secondIssueFirstFlow);
             testSubject.SelectedIssue = secondIssue;
             eventHandler.Verify(x => x(testSubject, It.Is((SelectionChangedEventArgs args) => args.SelectedFlow == secondIssueFirstFlow)), Times.Once());
             testSubject.SelectedFlow.Should().Be(secondIssueFirstFlow);
 
             eventHandler.VerifyNoOtherCalls();
         }
 
-        private static IAnalysisIssueFlowVisualization GetFlowWithLocation(out IAnalysisIssueLocationVisualization firstLocation)
+        [TestMethod]
```

Comment:
3 new tests that used to be in IssueSelectionServiceTests: null issue, issue without SLs, and issue with SLs.